code/datums/status_effects/gun_skill.dm

```diff
@@ -1,46 +1,75 @@
-/datum/status_effect/stacking/gun_skill
```

Comment:
Why change it from stacking ??

Reply:
Stacking is cringe because the effect is based on the number of stacks remaining, meaning it does almost nothing at low stacks. It also makes it harder to balance as high stacks make it impossible to hit a barn door.